docs/02_cleos/03_command-reference/get/account.md

```diff
@@ -1,20 +1,42 @@
+
+## Command
+cleos get account [OPTIONS] name [core-symbol]
+
+**Where**
+* [OPTIONS] = See Options in Command Usage section below.
+
+**Note**: The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Retrieves an account from the blockchain
+Retrieve an account from the blockchain
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos create key` command:
 
-## Positional Parameters
+### Positionals:
 - `name` _TEXT_ - The name of the account to retrieve
-- `core-symbol` _TEXT_ - The expected core symbol of the chain you are querying
+- `core-symbol` _TEXT_ - The expected core symbol of the chain you are querying - OPTIONAL
 
-## Options
+### Options
+- `-h,--help` Print this help message and exit
 - `-j,--json` - Output in JSON format
 
+## Requirements
+* Install the currently supported version of `cleos.`
+[[info | Note]] | The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
```

Comment:
The pipe | should start at the next line.

The cleos tool -> \`cleos\`

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.